core/src/test/java/org/elasticsearch/common/unit/FuzzinessTests.java

```diff
@@ -93,8 +93,12 @@ public void testParseFromXContent() throws IOException {
                 if (randomBoolean()) {
                     json = Fuzziness.AUTO.toXContent(jsonBuilder().startObject(), null).endObject();
                 } else {
+                    String auto = randomBoolean() ? "AUTO" : "auto";
+                    if (randomBoolean()){
+                        auto += ":" + randomIntBetween(0, 3) + "," + randomIntBetween(3, 10);
+                    }
                     json = jsonBuilder().startObject()
-                            .field(Fuzziness.X_FIELD_NAME, randomBoolean() ? "AUTO" : "auto")
+                            .field(Fuzziness.X_FIELD_NAME, auto)
                             .endObject();
                 }
```

Comment:
can you add more tests about making sure that we parse the low and high limits correctly?